reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md

```diff
@@ -868,33 +868,42 @@ The cmdlet does not return any output.
-  PowerShell responds to **the Enter-PSSession** command. You will be connected as soon as the
-  session is available. To cancel the `Enter-PSSession` command, press `CTRL+C`.
-* The **HostName** parameter set was included starting with PowerShell 6.0. It was added to provide
-  PowerShell remoting based on Secure Shell (SSH). Both SSH and PowerShell are supported on multiple
-  platforms (Windows, Linux, macOS) and PowerShell remoting will work over these platforms where
-  PowerShell and SSH are installed and configured. This is separate from the previous Windows only
-  remoting that is based on WinRM and much of the WinRM specific features and limitations do not
-  apply. For example WinRM based quotas, session options, custom endpoint configuration, and
-  disconnect/reconnect features are currently not supported. For more information about how to set
-  up PowerShell SSH remoting, see
-  [PowerShell Remoting Over SSH](/powershell/scripting/learn/remoting/ssh-remoting-in-powershell-core).
+To connect to a remote computer, you must be a member of the Administrators group on the remote
+computer. To start an interactive session on the local computer, you must start PowerShell with the
+**Run as administrator** option.
+
+When you use `Enter-PSSession`, your user profile on the remote computer is used for the interactive
+session. The commands in the remote user profile, including commands to add PowerShell modules and
+to change the command prompt, run before the remote prompt is displayed.
+
+`Enter-PSSession` uses the UI culture setting on the local computer for the interactive session. To
+find the local UI culture, use the $UICulture automatic variable.
+
+`Enter-PSSession` requires the `Get-Command`, `Out-Default`, and `Exit-PSSession` cmdlets. If these
+cmdlets are not included in the session configuration on the remote computer, the `Enter-PSSession`
+commands fails.
+
+Unlike `Invoke-Command`, which parses and interprets the commands before it sends them to the remote
+computer, `Enter-PSSession` sends the commands directly to the remote computer without
+interpretation.
+
+If the session you want to enter is busy processing a command, there might be a delay before
+PowerShell responds to **the Enter-PSSession** command. You will be connected as soon as the session
+is available. To cancel the `Enter-PSSession` command, press `CTRL+C`.
+
+The **HostName** parameter set was included starting with PowerShell 6.0. It was added to provide
+PowerShell remoting based on Secure Shell (SSH). Both SSH and PowerShell are supported on multiple
+platforms (Windows, Linux, macOS) and PowerShell remoting will work over these platforms where
+PowerShell and SSH are installed and configured. This is separate from the previous Windows only
+remoting that is based on WinRM and much of the WinRM specific features and limitations do not
+apply. For example WinRM based quotas, session options, custom endpoint configuration, and
```

Comment:
For example should either be followed by a colon or a comma. Colon would be more common but might look odd? Let me know what you think

Reply:
Fixed